src/game/input/mod.rs

```diff
@@ -15,9 +16,12 @@ pub struct TextInputPlugin;
 impl Plugin for TextInputPlugin {
     fn build(&self, app: &mut App) {
         app.add_systems(OnEnter(AppState::InGame), setup_text_input)
-            .add_systems(Update, (
-                listen_received_character_events.run_if(in_state(AppState::InGame)),
-                listen_keyboard_input_events.run_if(in_state(AppState::InGame))))
+            .add_systems(
+                Update,
+                (
+                    listen_received_character_events.run_if(in_state(AppState::InGame)),
+                    listen_keyboard_input_events.run_if(in_state(AppState::InGame))
+                ))
```

Comment:
I meant that it can be called on the tuple itself. doesn't really matter just wanted to let you know its possible
```
.add_systems(
                Update,
                (
                    listen_received_character_events,
                    listen_keyboard_input_events,
                )
                    .run_if(in_state(AppState::InGame)),
            )
                ```